test/e2e/tests/transaction/ens.spec.ts

```diff
@@ -59,17 +62,45 @@ describe('ENS', function (this: Suite) {
         },
       }));
 
-    await mockServer
-      .forPost(infuraUrl)
-      .withJsonBodyIncluding({ method: 'eth_call' })
-      .thenCallback(() => ({
-        statusCode: 200,
-        json: {
-          jsonrpc: '2.0',
-          id: '1111111111111111',
+    await mockServerJsonRpc(mockServer, [
```

Comment:
A sequence of mocks are required now for the updated ethersjs resolution process